ui/apps/platform/src/utils/textUtils.ts

```diff
@@ -16,7 +16,18 @@ export function pluralizeHas(len) {
     return len === 1 ? 'has' : 'have';
 }
 
+export function dedupeDelimtedString(value: string, delimiter = ','): string[] {
+    return Array.from(
+        new Set(
+            value.split(delimiter).map(
+                (v) => (typeof v === 'string' ? v.trim() : v) // type assertion to ensure string
+            )
+        )
+    );
+}
+
 export default {
     truncate,
     pluralizeHas,
+    dedupeDelimtedString,
```

Comment:
This seems like an obsolete idiom that is redundant with `export` above.

Reply:
This idiom allows for importing all the functions under one object name, similar to lodash, and use them like this: 

```
import textUtils from 'utils/textUtils';
...
const truncatedString = textUtils.truncate(originalString);
```

There also used to be a eslint rule "prefer-default-export", that didn't want you to export just a single names function. So, even if we never imported it lodash-style, having this kept eslint happy. I think that rule no longer applies in general.

I have removed this; we are not using it lodash-style anywhere in our code base.